ntbs-integration-tests/NotificationPages/ManualTestResultEditPagesTests.cs

```diff
@@ -54,8 +57,8 @@ public static IList<Notification> GetSeedingNotifications()
         public async Task PostNewManualTestResult_ReturnsSuccessAndAddsResultToTable()
         {
             // Arrange
-            var NotificationId = Utilities.DRAFT_ID;
-            var url = GetCurrentPathForId(NotificationId);
+            const int notificationId = Utilities.DRAFT_ID;
+            var url = GetCurrentPathForId(notificationId);
             var initialPage = await Client.GetAsync(url);
             var initialDocument = await GetDocumentAsync(initialPage);
```

Comment:
You might want to update to the `GetDocumentForUrl` helper from Laura here (and rest of the file)